src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -59,15 +66,19 @@
     private final MessagesConfigurator messagesConfigurator;
     private final ActionsConfigurator actionsConfigurator;
     private final BillPayConfigurator billpayConfigurator;
+    private final PocketsConfigurator pocketsConfigurator;
     private final LegalEntityService legalEntityService;
     private final AccountStatementsConfigurator accountStatementsConfigurator;
     private final PositivePayConfigurator positivePayConfigurator;
+    private final UserPresentationRestClient userPresentationRestClient;
+    private final PocketTailorActuatorClient pocketTailorActuatorClient;
 
     /**
```

Comment:
You can include our service name in the comment

Reply:
Done